go/client/chat_cli_rendering.go

```diff
@@ -76,16 +102,25 @@ func (v conversationListView) show(g *libkb.GlobalContext, myUsername string, sh
 	ui := g.UI.GetTerminalUI()
 	w, _ := ui.TerminalSize()
 
+	prefixNumOfBytes, err := ConversationAliasPrefixNumOfBytes(v.aliases(), 2)
+	if err != nil {
+		return err
+	}
+
 	table := &flexibletable.Table{}
-	for i, conv := range v {
+	for _, conv := range v {
+		indexCell := flexibletable.Cell{
+			Frame:     [2]string{"[", "]"},
+			Alignment: flexibletable.Right,
+			Content: flexibletable.SingleCell{
+				Item: string(MakeFullConversationAlias(conv.Info.Id).MustShorten(prefixNumOfBytes)),
+			},
+		}
```

Comment:
ConversationListView shows a list with conversation info and latest messages. ConversationInfoListView only shows the conversation info, and was used as part of the menu. The latter isn't used anymore now that the menu is gone; I didn't remove it because I thought we might want to be able to show full TLF names for users, maybe through a separate command.